app/controllers/api/event_hours_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Api
+  # Controller to expose Event Hours
+  class EventHoursController < Api::BaseController
+    before_action :set_event_date, only: %i[show index]
+    before_action :set_event_hour, only: %i[show]
+
+    # GET api/event_hours
+    # GET api/event_dates/:id/event_hours
+    def index
+      if params.key?(:event_date_id)
+        render json: ActiveModelSerializers::SerializableResource.new(
+          @event_date, event_hours: true, event_slots: true
+        ).as_json
+      else
+        render json: ActiveModelSerializers::SerializableResource.new(
+          EventHour.distinct.limit(100)
```

Comment:
Nice, sensible throttle until we have a feature that would want this kind of information